libs/application/templates/parental-leave/src/fields/InReviewSteps/index.tsx

```diff
@@ -104,6 +109,9 @@ const InReviewSteps: FC<React.PropsWithChildren<FieldBaseProps>> = (props) => {
               ? formatMessage(
                   parentalLeaveFormMessages.reviewScreen.adoptionDate,
                 )
+              : periods?.[0]?.firstPeriodStart ==
```

Comment:
```suggestion
              : periods?.[0]?.firstPeriodStart ===
```
Expected '===' and instead saw '=='. eslint